ibm_watson.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "concurrent-ruby", "~> 1.0"
   spec.add_runtime_dependency "eventmachine", "~> 1.2"
   spec.add_runtime_dependency "faye-websocket", "~> 0.11"
-  spec.add_runtime_dependency "http", "~> 4.1.0"
+  spec.add_runtime_dependency "http", "~> 4.4.0"
   spec.add_runtime_dependency "ibm_cloud_sdk_core", "~> 1.1.1"
```

Comment:
This would need to specify the latest core version if we merged it into the core.